<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-M23FVFN" failures="1" tests="4" name="TFP - Configurações - Bloqueios &gt; Sub Periodo" time="1588.758" errors="0" timestamp="30 nov. 2021 19:58:09 GMT">
  <testcase classname="com.timp.test.TFP.Configurações.Bloqueios.SubPeriodo.BloqueioSubPeriodoCriar" name="criar" time="328.505"/>
  <testcase classname="com.timp.test.TFP.Configurações.Bloqueios.SubPeriodo.BloqueioSubPeriodoEditar" name="editar" time="376.891"/>
  <testcase classname="com.timp.test.TFP.Configurações.Bloqueios.SubPeriodo.BloqueioSubPeriodoVer" name="ver" time="283.473"/>
  <testcase classname="com.timp.test.TFP.Configurações.Bloqueios.SubPeriodo.SubPeriodoFiltroId" name="criar" time="290.998">
    <failure type="java.lang.AssertionError" message="O registro não foi criado com sucesso expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: O registro não foi criado com sucesso expected [true] but found [false]
at com.timp.test.TFP.Configurações.Bloqueios.SubPeriodo.SubPeriodoFiltroId.criar(SubPeriodoFiltroId.java:44)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- criar -->
</testsuite> <!-- TFP - Configurações - Bloqueios > Sub Periodo -->
